Flow.Launcher.Core/Configuration/Portable.cs

```diff
@@ -127,7 +127,7 @@ public void CreateUninstallerEntry()
 
             using (var portabilityUpdater = NewUpdateManager())
             {
-                portabilityUpdater.CreateUninstallerRegistryEntry();
+                _ = portabilityUpdater.CreateUninstallerRegistryEntry();
```

Comment:
I think we should make CreateUninstallerEntry either `async Task` or `async void`. I am curious what this method is doing.

Reply:
`async void` needs to be [avoided](https://docs.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void), in fact I want to clean up some of those in the next step if I get a chance.

For this one, it just so happens that Squirrel's function is async, but we're not bothering to call it asynchronously. In fact, nothing in [`IPortable.cs`](https://github.com/Flow-Launcher/Flow.Launcher/blob/dev/Flow.Launcher.Core/Configuration/IPortable.cs) seems to be `async`.

I'm happy to create a follow up task for this, but changing the signature of the interface seems like a more invasive task than just removing the warning from the existing flow.

Reply:
if we didn't await the task, any exception thrown won't be caught. I wonder whether that may create some issue (previously I have been deeply hurt by this suppression).